source/components/panel/panel-docs.less

```diff
@@ -0,0 +1,26 @@
+
+.panel-demo {
+  height: 200px;
+  max-height: 400px;
+  min-height: 200px;
+  overflow: hidden;
+  resize: vertical;
+
+  hx-panel {
+    border: 1px dotted black;
+  }
+
+  hx-panelhead {
+    background-color: @purple-900;
```

Comment:
are the background colors approved by design? are these just used for examples and user can change?

Reply:
Background colors are purely for documentation. No need for design sign-off.